### DISCUSSION TITLE
M5 Stack Atom LED matrix with TA 3.0x

### DISCUSSION THREAD

**Discussion by Ja5-n** [josephdadams/TallyArbiter#547] [Q&A] [answered]
Hi folks
Once again thank you for an amazing product.
We just rolled out a 3 x Atom listener tally system using TA 2.x which worked brilliantly.
(using a gl-inet slate mobile router to manage the internal WiFi network for NDI and Tally signalling)

I'd now like to try to use the 3.0.1 TA code on my production machine but can't seem to find where the listener code might be for the Atom listener. Is there new code?

Jason

**Comment by mg-1999** [comment, marked as answer]
In this repo there is a folder called "listener clients" there you can find the new sourcecode for v3.

**Comment by Ja5-n** [comment]
Doh! I had (wrongly) assumed that was the 2.x code due to the date stamp. My bad.
So now, after the traditional amount of time fiddling with libraries, I have a working 3.x setup.

Thanks!

Jason

**Reply by gvcandyman** [reply]
Could you share what libraries you used? I'm having trouble with the Atom and StickC Plus 2 showing anything after uploading the firmware.
